datasets/consumer-finance-complaints/README.md

```diff
@@ -0,0 +1,179 @@
+| Submitted via   | How the complaint was submitted to the CFPB        | plain text        | This field is a categorical variable.        |
+| Date sent to company   | The date the CFPB sent the complaint to the company        | date & time        |         |
+| Company response to consumer   | This is how the company responded. For example, "Closed with explanation."        | plain text        | This field is a categorical variable.        |
+| Timely response?   | Whether the company gave a timely response        | plain text        | yes/no        |
+| Consumer disputed?   | Whether the consumer disputed the company’s response        | plain text        | YES/ NO/ N/A: The Bureau discontinued the consumer dispute option on April 24, 2017.        |
+| Complaint ID   | The unique identification number for a complaint        | number        |         |
+
+
+### Supported Tasks and Leaderboards
+
+Text Classification Tasks
+
+| Task      | Label Name | Description | SOTA |
+| ----------- | ----------- |----------- | ----------- |
+| Text Classification      | Product| Predict the related product of a complaint | N/A |
+
+| Task      | Label Name | Description | SOTA |
+| ----------- | ----------- |----------- | ----------- |
+| Text Classification      | Sub-Product| Predict the related sub product of a complaint | N/A |
+
+| Task      | Label Name | Description | SOTA |
+| ----------- | ----------- |----------- | ----------- |
+| Text Classification      | Tags | Predict whether a complaint has been made by someone elderly or a service person| N/A |
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+This dataset is a point in time extract of the database, the database increases in size every day
+
+### Data Fields
+
+[More Information Needed]
+
+### Data Splits
+
+This dataset only contains a TRAIN set - this can be further split into TRAIN, TEST and VALIDATE subsets with the datasets library
+
+## Dataset Creation
+
+### Curation Rationale
+
+Open sourcing customer complaints
+
+### Source Data
+
+https://cfpb.github.io/api/ccdb/
+
+#### Initial Data Collection and Normalization
+
+This database is maintained by the Consumer Financial Protection Bureau
+
+#### Who are the source language producers?
+
+Englisgh
+
+### Annotations
+
+#### Annotation process
+
+User submitted to the CFPB
+
+#### Who are the annotators?
+
+N/A
+
+### Personal and Sensitive Information
+
+All PII data has been anonymised
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+N/A
+
+### Discussion of Biases
+
+N/A
+
+### Other Known Limitations
+
+N/A
+
+## Additional Information
+
+### Dataset Curators
+
+https://cfpb.github.io/api/ccdb/
+
+### Licensing Information
+
+https://cfpb.github.io/source-code-policy/
```

Comment:
It sounds like the policy for source code of the Consumer Complaint Database API that is open source, is there some lincensing/terms of use for the data themselves ?